chapter1/web-search-agent/agent.py

```
@@ -269,6 +271,20 @@ def search_and_answer(self, user_question: str, max_iterations: int = 5) -> str:
                             "name": tool_call_name,
                             "content": tool_content
                         })
+                elif finish_reason == "length":
+                    # 输出预算（max_tokens）耗尽导致截断：返回已生成内容并明确标注，
```

Ruff (0.15.21)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)